src/lib/scene-internal.ts

```diff
@@ -841,13 +844,17 @@ export class SceneInternal implements Renderer {
     return this.hitTestOutputResults.subarray(0, sprites.length);
   }
 
-  doDraw() {
+  private doDraw() {
     // Initialize view if it hasn't been already.
     this.initView();
 
     const currentTimeMs = this.elapsedTimeMs();
 
-    this.drawCommand();
+    if (this.isViewInitialized) {
+      this.drawCommand();
+    } else {
+      console.warn('Skipping draw: view is not initialized');
```

Comment:
If the view hasn't been initialized, there's nothing to draw. This could be intentional. The API user may have an off-screen Scene that they intend to display later.

However, more often than not this is an error condition where the API user meant to show the Scene but hasn't added it to the DOM, or has given it a zero sized width or height (perhaps accidentally). So here we log the problem to the console so the API user can diagnose the situation.